checklist.bs

```diff
@@ -5,8 +5,10 @@ Level: 2
 Status: LD
 URL: https://wbcsd.github.io/pact-conformance-testing/checklist
 Mailing List: pact@wbcsd.org
-Editor: Prasad Tilloo (SINE Foundation), https://sine.foundation, prasad@sine.foundation
-Editor: Raimundo Henriques (SINE Foundation), https://sine.foundation, raimundo@sine.foundation
+Editor: Beth Hadley (WBCSD), https://www.wbcsd.org, hadley@wbcsd.org
```

Comment:
Could we actually list Arunav here instead of me (I just double checked with him), he will maintain this one going forward. 
Arunav Chakravarty (WBCSD), https://www.wbcsd.org, chakravarty@wbcsd.org